### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.5 - es. 3.16 (21 ottobre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#299] [Esercizi M-Z]
<img width="965" alt="immagine" src="https://github.com/user-attachments/assets/183aca47-b35d-4059-a5f2-da449463df7f">

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/19f08e4f-f812-40af-8a59-47ed452c1893)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/3c20c14f-81f2-4a73-85bb-ba6b7c043497)

**Comment by Andrii04** [comment]
![IMG_0720](https://github.com/user-attachments/assets/00c4a8ad-2820-4abd-ab62-160b141c99dd)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/a474cb5f-44de-4df5-b2a0-cacd87bb42e4)
